nucypher/blockchain/eth/actors.py

```diff
@@ -858,10 +862,35 @@ def locked_tokens(self, periods: int = 0) -> NU:
 
     @property
     def current_stake(self) -> NU:
+        """The total number of staked tokens, i.e., tokens locked in the current period."""
+        return self.locked_tokens(periods=0)
+
+    @property
+    def active_stakes(self) -> Generator[Stake, None, None]:
```

Comment:
IMO, it's clearer to annotate this as `Iterable[Stake]`, as discussed here (https://docs.python.org/3/library/typing.html#typing.Generator).

Reply:
:heavy_check_mark: